central/complianceoperator/v2/integration/datastore/datastore_impl_test.go

```diff
@@ -246,6 +246,66 @@ func (s *complianceIntegrationDataStoreTestSuite) TestGetComplianceIntegrations(
 	}
 }
 
+func (s *complianceIntegrationDataStoreTestSuite) TestGetComplianceIntegrationsView() {
+	testIntegrations := getDefaultTestIntegrations()
+	viewIntegrations := getDefaultTestIntegrationViews()
+	ids := s.addBaseIntegrations(testIntegrations)
+	for i, id := range ids {
+		testIntegrations[i].Id = id
+		viewIntegrations[i].ID = id
+	}
+
+	// Add some clusters
+	_, err := s.db.DB.Exec(context.Background(), "insert into clusters (id, name, status_providermetadata_cluster_type, type) values ($1, $2, $3, $4)", testconsts.Cluster1, "cluster1", 1, 1)
```

Comment:
Why not using the cluster store instead?

Reply:
Because a while back there were updates to the cluster store to make it use a cache.  So that increased the hassle of using the cluster store in tests outside the cluster datastore.  And since it was a simple insert, the extra hassle was not worth the effort, IMO.